docs/source/overview.rst

```diff
@@ -0,0 +1,305 @@
+.. _overview:
+
+========================
+Overview
+========================
+
+SkyPilot enables you to combine your cloud infra --- Kubernetes
+clusters, clouds and regions for VMs, and existing machines --- into a unified compute pool.
+
+.. image:: images/skypilot-abstractions-long-2.png
+    :align: center
+
+You can then run workloads on this pool in a unified interface, using these core abstractions:
+
+- Clusters
+- Jobs
+- Services
+
+These abstractions allow you to run all use cases in the AI lifecycle:
+Batch processing, development, (pre)training, finetuning, hparam sweeps, batch inference, and online serving.
+
+In SkyPilot, every workload benefits from:
+
+.. dropdown:: Unified execution on any cloud, region, and cluster
+
+    Regardless of how many clouds, regions, and clusters you have, you can use a unified interface
+    to submit, run, and manage workloads on them.
+
+    Your focus on the workload, and SkyPilot alleviates the burden of
+    dealing with cloud infra details and differences.
+
+.. dropdown:: Cost and capacity optimization
+
+    When launching a workload, SkyPilot will automatically choose the cheapest and most available infra choice in your search space.
+
+.. dropdown:: Auto-failover across infra choices
+
+    When launching a workload, you can give SkyPilot a search space of infra
+    choices --- as unrestricted or as specific as you like.
+
+    If an infra choice has no capacity,
+    SkyPilot automatically falls back to the next best choice in your infra search space.
+
+.. dropdown:: No cloud lock-in
+
+    Should you add infra choices (e.g., a new cloud, region, or cluster) in the future, your existing workloads can easily run on them.
+    No complex migration or workflow changes.
+    See the underlying :ref:`Sky Computing <sky-computing>` vision.
+
+.. _concept-dev-clusters:
+
+Clusters
+------------
+
+A *cluster* is SkyPilot's core resource unit: one or more VMs or Kubernetes pods in the same location.
+
+You can use ``sky launch`` to launch a cluster:
+
+.. tab-set::
+
+    .. tab-item:: CLI
+        :sync: cli
+
+        .. code-block:: console
+
+            $ sky launch
+            $ sky launch --gpus L4:8
+            $ sky launch --num-nodes 10 --cpus 2+ --memory 2+ --down
```

Comment:
nit: since we have `--down` but no job specified, won't this immediately auto-down the launched cluster? Could move `--down` to the example with cluster.yaml.